components/addEditPurchase.js

```diff
@@ -155,7 +158,13 @@ const AddEditPurchase = ({ purchase }) => {
 
         {companyId && selectedListType && (
           <>
-            <AddItemsInPo companyId={companyId} type={selectedListType} setData={setData} data={data} />
+            <AddItemsInPo
+              isEdit={purchase && true}
```

Comment:
```suggestion
              isEdit={purchase && true}
```
Why are we sending this as true, we can utilize default props for this

Reply:
we just need isEdit true OR false if purchase is present then edit mode is true otherwise false